### PR TITLE
Rename dask service port from `comm` to `tcp-comm` #patch

### PR DESCRIPTION
**Pull request by bstadlbauer**
# TL;DR
This PR renames the port of the `dask` plugin's scheduler service from `comm` to `tcp-comm` as this helps with routing within istio ([docs](https://istio.io/v1.0/docs/setup/kubernetes/spec-requirements/)). 
In addition, the `dask-kubernetes` controller defaults to this in certain places (e.g. [here](https://github.com/dask/dask-kubernetes/blob/066825f208e03030d0a68be94b7877507876bd88/dask_kubernetes/common/networking.py#L20))

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

### CODE REVIEW

**Comment by hamersaw** [comment]
@bstadlbauer can we merge master into this? It should fix the checks now.

**Comment by bstadlbauer** [comment]
@hamersaw Done ✅

**Review by hamersaw — APPROVED** [review]
